gensim/models/fasttext.py

```diff
@@ -1264,14 +1263,14 @@ def save(self, *args, **kwargs):
         kwargs['ignore'] = kwargs.get('ignore', ignore_attrs)
         super(FastTextKeyedVectors, self).save(*args, **kwargs)
 
-    def get_vector(self, word, use_norm=False):
+    def get_vector(self, word, norm=False):
```

Comment:
`use_norm` was an unfortunate choice of parameter-name which I believe dated back to #1777,  & I considered changing it, but didn't want to have to change all uses elsewhere & add another compatibility-burden on users. 

But if you're entertaining breaking changes:

(1) not sure `norm` is a good name either - it makes it sound like you're requesting the vector's (scalar) norm/length;
(2) even supporting this via an optional extra parameter on a dual-use method strikes me as silly/opaque, too. Instead, have `__getitem__` be the implementation of the default behavior, add a separate explicitly-named method for the unit-normed vector, like perhaps `get_unit_vector(key_or_int)`, no optional parameters;
(3) more aggressively, maybe let users **toggle** the model to give either unit-vecs, or raw-vecs, as the default `__getitem__` behavior, but have both `get_unit_vector()` and `get_raw_vector()` alt options for when a caller needs to specify. Then the desire you've expressed elsewhere to have unit-normed access, via plain dict-bracket-accesses, is available when toggled-on, but the raw data is never destroyed, preventing a class of errors/regrets. I'd put this thought aside as a somewhat complicated "maybe in the future" idea, but if you're OK to break `get_vector()` signature, & want to prioritize unit-normed access, maybe it's worth making that change now.

Reply:
TBH I find the idea of producing a new standalone (normed) KV instance cleaner than toggles.

I'll merge here lest we get a monster PR again, let's revisit this in one of your other PRs.